obspy/imaging/waveform.py

```diff
@@ -1300,6 +1308,10 @@ def __sect_init_plot(self):
             else:
                 raise NotImplementedError("sect_orientiation '%s' is not "
                                           "valid." % self.sect_orientation)
+            if self.wiglets:
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data > self._tr_offsets_norm[_tr], facecolor='blue')
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data < self._tr_offsets_norm[_tr], facecolor='red')
```

Comment:
I know, right now `WaveformPlotting` really is an ugly monster and so far not very user-friendly in terms of controlling certain aspects (and in many places rather messy and implicit/intransparent by fishing tons of things out of `kwargs`)..

but.. how about starting to make it a better place.. ;-) by making those two color specs class attributes? That way, users could later have custom colors (or even just one of the two -- which is a rather typical use case) without having to hack the obspy source code..

```python
class WaveformPlotting(object):
    """
    """
    wiglet_color_1 = "blue"
    wiglet_color_2 = "red"

[...]

        if self.wiglets:
            if self.wiglet_color_1 is not None:
                fillfun(time, data, self._tr_offsets_norm[_tr], where=data > self._tr_offsets_norm[_tr], facecolor=self.wiglet_color_1)
[...]
```

Users could than change it at run time like..
```python
from obspy.imaging.waveform import WaveformPlotting
WaveformPlotting.wiglet_color_1 = "k"
WaveformPlotting.wiglet_color_1 = None

from obspy import read
read().plot(type="section")
```